samples/apps/copilot-chat-app/webapp/src/Constants.ts

```diff
@@ -20,7 +20,7 @@ export const Constants = {
     bot: {
         profile: {
             id: 'bot',
-            fullName: 'Copilot',
+            fullName: 'Eva',
```

Comment:
undo?

(But, given how many places you had to change to change the name of the app, I'm thinking we might need to make that simpler so users can make that change in a single place. I'll add an issue for that!)

Reply:
Yeah... I've gone back and forth changing that in all those places so many times and am embarrassed it made it into the PR :(